models/cycle_gan_semantic_mask_model.py

```diff
@@ -59,7 +59,17 @@ def modify_commandline_options(parser, is_train=True):
     
     def __init__(self, opt):
         BaseModel.__init__(self, opt)
-        
+        if not hasattr(opt, 'disc_in_mask'):
```

Comment:
This could be moved to default values of argument parser instead

Reply:
It's because these parameters are not available from the inference command line options. I could add them but since we actually don't want to set them, I'd rather leave them as is for the moment.